deeplabcut/pose_estimation_tensorflow/superanimal_configs/superquadruped.yaml

```diff
@@ -0,0 +1,1632 @@
+all_joints:
```

Comment:
I think we should remove the PAF graphs and set partaffinityfield_predict: false in both configs.

Reply:
Yes I did that in the next PR